lib/models/Quote.js

```diff
@@ -15,8 +15,14 @@ class Quote {
     return Number(rows[0].count);
   }
 
-  static async insert() {
+  static async insert({ episode_id, character_id, detail }) {
     // implement insert to add new quote
+    const { rows } = await pool.query(
+      'INSERT INTO quotes (episode_id, character_id, detail) VALUES ($1, $2, $3) RETURNING *',
+      // we add $1, $2, $3 to specify we are looking for the first three values in the object, which is episode_id, character_id and detail
+      [episode_id, character_id, detail]
```

Comment:
Don't forget to add in episode_id in the constructor so that when posting to quotes, episode_id is also returned.